era5_monthly_averages_single_level_1979_2024/process_downloads.jl

```diff
@@ -1,6 +1,42 @@
 using NCDatasets
 using Dates
 
+"""
+    merge_zipped_download(input_path)
+
+Merges NetCDF files that were downloaded in a zipped format. The zipped datasets are expected
+to share the same dimensions.
+"""
+function merge_zipped_download(input_path)
+    output_name = splitext(input_path)[1]
+    run(`unzip -o $input_path -d $output_name`)
+    split_file_names = readdir(output_name)
+    split_file_names = filter(x -> splitext(x)[2] == ".nc", split_file_names)
```

Comment:
I am not sure what is being accomplished here, but it might be easier to use `endswith` instead if you just need to check which files end with ".nc"?